containers/runner/launch

```diff
@@ -29,18 +29,22 @@ Options:
  -j, --jobs N                         Run N jobs in parallel (default: $(nproc))
  -t, --testtype TYPE                  Only run TYPE tests
  -s, --skip-testtypes TYPE[,TYPE..]   Don't run tests with given types
+ --daily-copr TOKEN_FILE              Download and use daily COPR boot.iso instead of rawhide's
```

Comment:
Not sure if we want to named it ``copr``. It seems to me like an implementation detail. Maybe having more generic name ``daily-build-iso`` would be better?

Reply:
Names are always difficult! I wanted to point out that it's not just a daily Rawhide iso. But I'm fine with calling the latter "rawhide" and this one "daily".

Reply:
Renamed to `--daily-iso` as per our IRC discussion.